src/event/containers/PastEvents.tsx

```diff
@@ -46,12 +42,11 @@ interface PastEventsContainerProps {
   ];
   fetchAttendance: Function;
   fetchPastEvents: Function;
-  updateTimeframe: Function;
-  timeframe: string;
 }
 
 const PastEventsContainer: React.FC<PastEventsContainerProps> = (props) => {
-  const { auth, events, attendance, fetchAttendance, fetchPastEvents, timeframe, updateTimeframe } = props;
+  const { auth, events, attendance, fetchAttendance, fetchPastEvents } = props;
+  const [timeframe, setTimeframe] = useState(getCurrentYear()?.name ?? 'All Time');
```

Comment:
out of scope i think: would be nice to use the similar typing [that `ucsd-quarters-years` uses](https://github.com/acmucsd/ucsd-quarters-years/blob/main/src/bounds.ts#L12) - since we do use that here.